frame/elections/CHANGELOG.md

```diff
@@ -4,6 +4,17 @@ All notable changes to this crate will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this crate adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## [5.0.0] - UNRELEASED
```

Comment:
Cool that you keep this maintained. As you see, we don't quite do it yet, but good practice.